pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -394,3 +394,7 @@ func (current *Revision) CheckImmutableFields(ctx context.Context, og apis.Immut
 
 	return nil
 }
+
+func compareSpec(original *Revision, current *Revision) (string, error) {
```

Comment:
not sure how much this saves us in readability or anything.

Reply:
Removed.